### ISSUE TITLE
Could not find the config file and checkpoint for ConvNext-S Mask R-CNN 3xMS

### ISSUE THREAD

**Issue by HashmatShadab** [MzeroMiko/VMamba#118]
Can you please let me know where I can find the config/checkpoint for  ConvNext-S Mask R-CNN 3xMS reported in the paper(Table 3). The one in the config is cascaded Mask RCNN

**Comment by MzeroMiko** [comment]
sorry, I am busy these days.

Do you mean this file? This file may be generated combining 3x swin and convnext.

https://github.com/MzeroMiko/VMamba/blob/main/detection/configs/convnext/mask-rcnn_convnext-t-p4-w7_fpn_amp-ms-crop-3x_coco.py

**Comment by HashmatShadab** [comment]
I can't find the checkpoints for the model

**Comment by MzeroMiko** [comment]
> Thanks. But, I can't find the checkpoints for the model ConvNext-S model with 3x schedule

Yes, we did not release that checkpoint. We may release it in the future.
